### PR TITLE
8328244: Convert javax/swing/JSlider/6742358/bug6742358.java applet test to main

### PR DESCRIPTION
**Pull request by kumarabhi006**
Conversion of manual applet test to main based using PassFailJFrame manual framework

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8328244](https://bugs.openjdk.org/browse/JDK-8328244): Convert javax/swing/JSlider/6742358/bug6742358.java applet test to main (**Bug** - P4)

### Reviewers
 * [Prasanta Sadhukhan](https://openjdk.org/census#psadhukhan) (@prsadhuk - **Reviewer**) ⚠️ Review applies to [976964f0](https://git.openjdk.org/jdk/pull/18321/files/976964f04510710259279d7a5dfd067c9b86d37d)
 * [Tejesh R](https://openjdk.org/census#tr) (@TejeshR13 - Committer)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/18321/head:pull/18321` \
`$ git checkout pull/18321`

Update a local copy of the PR: \
`$ git checkout pull/18321` \
`$ git pull https://git.openjdk.org/jdk.git pull/18321/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 18321`

View PR using the GUI difftool: \
`$ git pr show -t 18321`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/18321.diff">https://git.openjdk.org/jdk/pull/18321.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/18321#issuecomment-1998995778)

### CODE REVIEW

File: test/jdk/javax/swing/JSlider/bug6742358.java [hunk 1]
```diff
@@ -24,34 +24,41 @@
 /* @test
```
Comment (prsadhuk) [inline test/jdk/javax/swing/JSlider/bug6742358.java:24]: start @test at next line

**Review by prsadhuk — APPROVED** [review]

File: test/jdk/javax/swing/JSlider/bug6742358.java [hunk 2]
```diff
@@ -21,37 +21,45 @@
  * questions.
  */
 
-/* @test
+/*
+ * @test
  * @bug 6742358
  * @summary MetalSliderUI paint wrong vertical disabled filled JSlider for DefaultMetalTheme
```
Comment (TejeshR13) [inline test/jdk/javax/swing/JSlider/bug6742358.java:27]: Summary sentence could be re-structured.

**Review by TejeshR13 — COMMENTED** [review]
Line column limit exceeds in `createSlider()` method, can correct those.

**Review by TejeshR13 — APPROVED** [review]

**Pull request closed without merging**